tests/typings/sync/sync.ts

```diff
@@ -2,7 +2,7 @@ import allure from '@wdio/allure-reporter'
 
 // browser
 browser.pause(1)
-const waitUntil: boolean = browser.waitUntil(() => true, 1, '', 1)
+const waitUntil: Promise<boolean> = browser.waitUntil(() => true, 1, '', 1)
```

Comment:
sync mode has nothing to do with promises!

Reply:
Got it, need to figure out the dark magic of sync/async conversion in this library.

At the moment it is clear to me that this function is asynchronous, but I see that you use Fibers to steam line it for test-runners.

Is sync version only applicable to test runner and standalone is always async?